src/io/flutter/FlutterInitializer.java

```diff
@@ -14,6 +14,7 @@
 import com.intellij.openapi.actionSystem.AnActionEvent;
 import com.intellij.openapi.application.ApplicationInfo;
 import com.intellij.openapi.application.ApplicationManager;
+import com.intellij.openapi.diagnostic.Logger;
```

Comment:
Why add this?

Reply:
oops - that was cruft left over from an experiment